### PR TITLE
Weekly #5: February

### PR DESCRIPTION
**Pull request by RichardLitt**
Initiate the Feb 1st weekly.

Please add stuff here if anything happens this week!

### CODE REVIEW

**Comment by RichardLitt** [comment]
Here is the draft. It is super late (apologies!), and I don't have much content for it. Can you all help out with this?

File: published/005-2016-Feb-1.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [February 1](//github.com/ipfs/pm/issues/88) sprint:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+	- [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated
+	- [@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs
```
Comment (ghost) [inline published/005-2016-Feb-1.md]: [@lgierth](//github.com/lgierth) improved the automated [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs, which is now named `ipfs/go-ipfs`. Give it a try: `docker run -i --net=host ipfs/go-ipfs`.

The new image is built automatically for every commit, and has automatic tags for the coming releases. It's also a few MB smaller, and doesn't require mounting a volume for the IPFS repository anymore. Instead, if there's no volume mounted, it'll generate an ephemeral identity and configuration which will be lost when the container exits. This is ideal for testing or just trying something out real quick. Images for go-ipfs 0.3.x are not provided, because the Dockerfile shipped with these version isn't suitable for Docker Hub automatic builds.

File: published/005-2016-Feb-1.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+	- [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated
+	- [@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs 
+	- [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
+* [**distributions**](//github.com/ipfs/distributions)
+	- [dist.ipfs.io](http://dist.ipfs.io/) now has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to @dignifiedquire
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
```
Comment (ghost) [inline published/005-2016-Feb-1.md]: That PR also includes libp2p doesn't it? Wanna write a few words @whyrusleeping? :)

File: published/005-2016-Feb-1.md (hunk 1 above)
Comment (ghost) [inline published/005-2016-Feb-1.md]: Updated the docker image comment ^

File: published/005-2016-Feb-1.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Here are some of the highlights for the [February 1](//github.com/ipfs/pm/issues/88) sprint:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+	- [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated
+	- [@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs 
+	- [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
+* [**distributions**](//github.com/ipfs/distributions)
+	- [dist.ipfs.io](http://dist.ipfs.io/) now has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to @dignifiedquire
```
Comment (dignifiedquire) [inline published/005-2016-Feb-1.md]: Signed releases are not shipped yet, still work in progress

**Comment by RichardLitt** [comment]
@eminence want to throw me the contribs?

**Comment by eminence** [comment]
```
Contributors for this period (2016-02-01T12:00:01 to 2016-02-08T12:00:01):
* ARezaK
* Christian Couder
* David Dias
* Friedel Ziegelmayer
* Jakub Sztandera
* Jeromy Johnson
* Juan Benet
* Lars Gierth
* Richard Littauer
* Stephen Whitmore
* Thomas Gardner
* greenkeeperio-bot
```

File: published/005-2016-Feb-1.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+  - [@chriscool](//github.com/chriscool) rewrote [the build documentation for Windows](//github.com/ipfs/go-ipfs/blob/master/docs/windows.md), which enabled [@slothbag](//github.com/slothbag) to get Windows builds working again with minimal fuss.
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). THis will be coming soon.
```
Comment (dignifiedquire) [inline published/005-2016-Feb-1.md]: Typo: T**h**is will be coming soon

**Comment by jbenet** [comment]
I will review this tonight. we should ship this soon as it is... march already.

File: published/005-2016-Feb-1.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+  - [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: sounds pretty odd. maybe "@noffle improved FUSE connection termination."

File: published/005-2016-Feb-1.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+  - [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated.  
+  - [@noffle](//github.com/noffle), [@whyrusleeping](//github.com/whyrusleeping) and [@chriscool](//github.com/chriscool) have been doing lots to make our tests greener. This'll let us get PRs in faster because we won't need to wonder whether our CI is lying to us or not. This gives us more confidence about the code we ship, and allows us to ship more often.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: There are ways to say this that give context accurately. The current language is inaccurate (CI isn't lying, the tests are wrong) and makes the CI or testing work look worse than it is. This is a common thing many teams struggle with and must fix. maybe this:
- @noffle, @whyrusleeping and @chriscool have been improving the test suite, fixing flaky tests which used to fail inaccurately. This will let us merge PRs faster, as CI should correctly categorize success and failure.

File: published/005-2016-Feb-1.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+  - [@noffle](//github.com/noffle) also made progress toward getting `go-ipfs` compatible with vanilla `go get` installation. Hope to have more tangible results to show by end of the month.
+  - [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md:21]: :+1:

File: published/005-2016-Feb-1.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+  - [@lgierth](//github.com/lgierth) improved the automated [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs, which is now named `ipfs/go-ipfs`. Give it a try: `docker run -i --net=host ipfs/go-ipfs`. The new image is built automatically for every commit, and has automatic tags for the coming releases. It's also a few MB smaller, and doesn't require mounting a volume for the IPFS repository anymore. Instead, if there's no volume mounted, it'll generate an ephemeral identity and configuration which will be lost when the container exits. This is ideal for testing or just trying something out real quick. Images for go-ipfs 0.3.x are not provided, because the Dockerfile shipped with these version isn't suitable for Docker Hub automatic builds.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: this is an excellent update. :+1: :clap: bravo. Enough context, actionable way to try, reasoning, etc.

File: published/005-2016-Feb-1.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+  - [@chriscool](//github.com/chriscool) rewrote [the build documentation for Windows](//github.com/ipfs/go-ipfs/blob/master/docs/windows.md), which enabled [@slothbag](//github.com/slothbag) to get Windows builds working again with minimal fuss.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md:22]: great news!

File: published/005-2016-Feb-1.md [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+  - [@lgierth](//github.com/lgierth) posted a writeup on the [migration from 0.3.x to 0.4.0](https://ipfs.io/blog/9-v04x-migration/).
+* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after four months. This reflected a ton of work from [@mildred](//github.com/mildred), [@findkiko](//github.com/findkiko), and [@jbenet](//github.com/jbenet). The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: This was mostly work between @mildred and I. @findkiko fixed a minor thing. But many others not mentioned contributed much more in comments and so on. Maybe:
- [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after months of thoughtful design. The bulk of the work was achieved by [@mildred](//github.com/mildred) and [@jbenet](//github.com/jbenet), with lots of comments and design opinions from many other contributors. The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed. Stay tuned for more information in the future.

File: published/005-2016-Feb-1.md [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs)
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: i agree with reducing repetition but the categories dont work as bullets, right now its very easy to get lost in the bullet levels. either make them larger (proper headings) or return to the original `- (topic) message` format, even if repetitive.

File: published/005-2016-Feb-1.md [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+  - [@lgierth](//github.com/lgierth) posted a writeup on the [migration from 0.3.x to 0.4.0](https://ipfs.io/blog/9-v04x-migration/).
+* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after four months. This reflected a ton of work from [@mildred](//github.com/mildred), [@findkiko](//github.com/findkiko), and [@jbenet](//github.com/jbenet). The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed.
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: I think we should call attention to things if
1. it is done and people should know about
2. there is a call to action -- opinions are needed.
3. it is specially useful to draw attention to something in the weekly.

if it's (3). ok. but if none of these, then be wary, because having this sort of line here -- "person X worked on Y" -- implies this list is more comprehensive as a work report than it is;  it is very much not. lots of other work has been going on that is not directly featured and shouldn't be.

File: published/005-2016-Feb-1.md [hunk 13]
```diff
@@ -0,0 +1,209 @@
...
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: this is a bigger deal, it deserves an update as comprehensive as @lgierth's above on the docker image. draw attention to this being a big deal, describe to users why it is so, describe what has changed and give a sense why this was needed.

Comment (jbenet) [inline published/005-2016-Feb-1.md]: the ipns tool is a separate item altogether, not under gx heading. link to it too.

File: published/005-2016-Feb-1.md [hunk 14]
```diff
@@ -0,0 +1,209 @@
...
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: again, like above, dont draw attention to just work. 

in this particular case, there's something interesting to note, for example:
- @RichardLitt improved the `gx` README document to better explain its goals, and help people get started. Take a look and give us feedback through an issue if you think anything could be improved.

File: published/005-2016-Feb-1.md [hunk 15]
```diff
@@ -0,0 +1,209 @@
...
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
+* [**js-ipfs**](//github.com/ipfs/js-ipfs) 
+	- Thanks to @diasdavid, `object features` now works, with tests and all.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: `object features` is vague. maybe "the dag object manipulation commands"

File: published/005-2016-Feb-1.md [hunk 16]
```diff
@@ -0,0 +1,209 @@
...
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
+* [**js-ipfs**](//github.com/ipfs/js-ipfs) 
+	- Thanks to @diasdavid, `object features` now works, with tests and all. 
+	- As well, thanks to [@dignifiedquire](//github.com/dignifiedquire)'s efforts, the js-ipfs API now returns promises if there isn't a specified callback, allowing both major methods used by the Javascript community to work equally well.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: @dignifiedquire @diasdavid oh my god, please do not mix and match these. this not a comment for this PR, but please revisit this. If I learned one thing from touching js promises and callbacks it's do not mix them, have different functions (different api objects) for them.

Comment (jbenet) [inline published/005-2016-Feb-1.md]: please discuss this elsewhere, i should've mentioned this at the source but i;m short on time. carry on conversation elsewhere and tag me, i'll follow later.

File: published/005-2016-Feb-1.md [hunk 17]
```diff
@@ -0,0 +1,209 @@
...
+* [**registry-mirror**](//github.com/diasdavid/registry-mirror) 
+	- [@diasdavid](//github.com/diasdavid) merged registry-static into registry-mirror and removed all of the unnecessary stuff [(this was huge)](https://github.com/diasdavid/registry-mirror/pull/33).
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: this line leaves me thinking "wat?".  How about:
- @diasdavid improved registry-mirror performance and robustness by removing the dependency on registry-static. The few necessary parts were copied in. This is a huge step in reliability and performance cloning the registry with registry-mirror.

File: published/005-2016-Feb-1.md [hunk 18]
```diff
@@ -0,0 +1,209 @@
...
+* [**station**](//github.com/ipfs/station)
+  - [@dignifiedquire](//github.com/dignifiedquire) fixed drag and drop
+    file uploading, and some dependency issues. So go ahead and try it
+    out.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: great! short, sweet, call to action.

File: published/005-2016-Feb-1.md [hunk 19]
```diff
@@ -0,0 +1,209 @@
...
+* [**ipfs-geoip**](//github.com/ipfs/ipfs-geoip)
+  - [@dignifiedquire](//github.com/dignifiedquire) rewrote the generation
+    script and cleaned up the code so that now the data is consistently
+    reproducable and fully stored on IPFS. This ensures that geoip lookups
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: "reproducible"

File: published/005-2016-Feb-1.md [hunk 20]
```diff
@@ -0,0 +1,209 @@
...
+    over IFPS will work in all future releases.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: @dignifiedquire they should work in previous releases too. we should be able to regenerate the exact data. the database has versions. so does the codebase.

File: published/005-2016-Feb-1.md [hunk 21]
```diff
@@ -0,0 +1,209 @@
...
+* [**fs-repo-migrations**](//github.com/ipfs/fs-repo-migrations)
+  - [@chriscool](//github.com/chriscool) Improved fs-repo-migrations tests by checking that the objects are the same and checking that garbage collection does not change anything. He also cleaned up sharness tests.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: this sounds exhaustively odd (to specific). how about:
- @chriscool improved important tests for fs-repo-migrations -- tests verify more edge cases when migrating forward and backward, through various sample workloads.

File: published/005-2016-Feb-1.md [hunk 22]
```diff
@@ -0,0 +1,209 @@
...
+* [**http-api-spec**](//github.com/ipfs/http-api-spec)
+  - @RichardLitt finished logging all existing `ipfs` commands for the HTTP API Spec. This means that if you have any questions about how the HTTP API should work, you can see them either in master or as open PRs to that issue. One step closer to a fully functional API...
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: solid update. maybe make the call to action stronger: "If you are interested in how the HTTP API should work, or have any specific questions, please see the current version (link to it), and explore the open PRs to that issue." or something

File: published/005-2016-Feb-1.md [hunk 23]
```diff
@@ -0,0 +1,209 @@
...
+* [**community**](//github.com/ipfs/community)
+  - @RichardLitt proposed using the new [GitHub Templates]() for IPFS. What do you think about this idea? Chime in [here](https://github.com/ipfs/community/issues/108).
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: "github templates" no link. (and great idea)

File: published/005-2016-Feb-1.md [hunk 24]
```diff
@@ -0,0 +1,209 @@
...
+  - We now have [a list of community events](https://github.com/ipfs/community#events-calendar) for IPFS on the ipfs/community README. Do you have any that you'd like to add? Check out past events, too.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: excellent! I think this community stuff should be under its own main `## Community` heading.

File: published/005-2016-Feb-1.md [hunk 25]
```diff
@@ -0,0 +1,209 @@
...
+* [**ipfs-hyperlog**](//github.com/noffle/ipfs-hyperlog) - @noffle launched am IPFS-compatible Merkle DAG that replicates based on scuttlebutt logs and causal linking. `ipfs-hyperlog` is a drop-in replacement for [@mafintosh](https://github.com/mafintosh)'s [`hyperlog`](https://github.com/mafintosh/hyperlog). Its key difference is that it creates a Merkle DAG that is *binary compatible* with IPFS objects. This means any node of any DAG built using ipfs-hyperlog can be replicated to and from the IPFS network as well!
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: maybe emphasize at the beginning that this is a fork of the project? it makes sound like this module was new. maybe:
- @noffle built ipfs-hyperlog, and ipfs-compatible version of hyperlog, a dag that ...

File: published/005-2016-Feb-1.md [hunk 26]
```diff
@@ -0,0 +1,209 @@
...
+
+## Work in Progress
+
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: lots of the stuff listed above was WIP too. so either consolidate that here, or just get rid of this WIP heading, and fold the logo work above

File: published/005-2016-Feb-1.md [hunk 27]
```diff
@@ -0,0 +1,209 @@
...
+* **Logo**: [@Kubuxu](//github.com/Kubuxu) worked on a new [IPFS logo](https://ipfs.io/ipfs/QmTgtbb4LckHaXh1YhpNcBu48cFY8zgT1Lh49q7q7ksf3M/). Check it out.
+
+## Community
+
+* **Conferences**: If you have any suggestions, there is a [new issue](https://github.com/ipfs/community/issues/105) where you can submit your proposals. The IPFS community will see if we can get to that conference and have a presence there.
+
+* [**Marmot Image Checker**](https://github.com/eris-ltd/marmot): This takes an image, asks the Google Cloud Vision API for three descriptions of the image, and compares those descriptions to a chosen list of words. If there is match, the image is added to the [toadserver](https://github.com/eris-ltd/toadserver), a smart contract and IPFS-Backed download server. [From Twitter](https://twitter.com/asciinema/status/701730719589126146)
+* [**asciienema**](https://github.com/asciinema/asciinema/releases/tag/v1.2.0) now allows playback on IPFS. [From Twitter](https://twitter.com/asciinema/status/701730719589126146)
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: awesome! :)

**Comment by jbenet** [comment]
let's make sure in CR/discussion here in this repo to escape text changes that include names of people not in our team, that way we don't tag them + subscribe them to listen in to a bunch of annoying CR on this :)

File: published/005-2016-Feb-1.md [hunk 28]
```diff
@@ -0,0 +1,209 @@
...
+* [Introduction to IPFS](https://medium.com/@ConsenSys/an-introduction-to-ipfs-9bba4860abd0#.48ki3fymw): [@ChrisLundkvist](https://twitter.com/ChrisLundkvist) and [@ConsenSysAndrew](https://twitter.com/ConsenSysAndrew) wrote a fantastic write up about IPFS, from a less technical perspective.
```
Comment (jbenet) [inline published/005-2016-Feb-1.md]: the write up is very technical, it's only the beginning that isn't. also, let's write a bit more about this, this was a big deal post that our friends wrote on our behalf. something like:

```
- Our friends and users at ConsenSys wrote an excellent "Introduction to IPFS", which starts with a less-technical preface, and then dives deep into a full explanation of how the IPFS object model works. It walks through multiple examples, including directory structures, version control systems, and blockchains. This is a great post to familiarize yourself with how the low level IPFS objects work, with graph visualizations and in-depth explanations. It also features a fantastic cover image! Thanks @ChrisLundkvist and @ConsenSysAndrew!

![](https://cdn-images-1.medium.com/max/2000/1*czZJ7mvEAqL4wNAg-jt9Ow.jpeg)
```

File: published/005-2016-Feb-1.md (hunk 20 above)
Comment (dignifiedquire) [inline published/005-2016-Feb-1.md]: As I said before I was unable to recover the previous data.

File: published/005-2016-Feb-1.md (hunk 16 above)
Comment (dignifiedquire) [inline published/005-2016-Feb-1.md]: This was very much welcomed, see https://github.com/ipfs/js-ipfs-api/pull/200 for feedback from others. Also already multiple modules depend on this work now, so it's not sth we can easily take back now.

File: published/005-2016-Feb-1.md (hunk 25 above)
Comment (hackergrrl) [inline published/005-2016-Feb-1.md]: Further nitpicking: "an ipfs-compatible **fork** of hyperlog"

File: published/005-2016-Feb-1.md (hunk 5 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Replaced.

File: published/005-2016-Feb-1.md (hunk 6 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Thanks; that is better.

File: published/005-2016-Feb-1.md (hunk 8 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Well done, @lgierth!

File: published/005-2016-Feb-1.md (hunk 10 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Works for me. Fixed.

File: published/005-2016-Feb-1.md (hunk 11 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Working on using subheadings. I think it looks better. Will show you when I update the rest of this.

File: published/005-2016-Feb-1.md (hunk 12 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Agreed.

File: published/005-2016-Feb-1.md (hunk 14 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Thanks. I'll use this text, and I see what you did there.

File: published/005-2016-Feb-1.md (hunk 16 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: I pinged you in that PR, @jbenet -- it is hard to discuss it when there were no opposing view points. Would love to get your opinion on it, because we could learn from your experience there.

File: published/005-2016-Feb-1.md (hunk 17 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Much better. Thank you.

File: published/005-2016-Feb-1.md (hunk 20 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: @jbenet Should we open an issue to see if we can get it, somehow?

File: published/005-2016-Feb-1.md (hunk 22 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Fixed.

File: published/005-2016-Feb-1.md (hunk 23 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Link added.

File: published/005-2016-Feb-1.md (hunk 24 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Moved.

File: published/005-2016-Feb-1.md (hunk 25 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Switched to combined suggestion. Thanks.

File: published/005-2016-Feb-1.md (hunk 26 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Removed WIP.

File: published/005-2016-Feb-1.md (hunk 28 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: This image is so good - I think we should move this paragraph to the top, above Updates, just to highlight it. What do you think?

File: published/005-2016-Feb-1.md (hunk 13 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: @whyrusleeping gave me some great copy on it. Updated.

**Comment by RichardLitt** [comment]
@jbenet Incorporated all of your feedback, and pushed. I think it is ready to ship. Confirmation?

This image from that article is so good - I think we should move this paragraph to the top, above Updates, just to highlight it. What do you think?

File: published/005-2016-Feb-1.md (hunk 20 above)
Comment (RichardLitt) [inline published/005-2016-Feb-1.md]: Linked! https://github.com/ipfs/ipfs-geoip/issues/8

**Comment by jbenet** [comment]
> This image from that article is so good - I think we should move this paragraph to the top, above Updates, just to highlight it. What do you think?

yeah agreed!

**Comment by jbenet** [comment]
@RichardLitt this is great, LGTM, thanks!
